reference/7.2/Microsoft.PowerShell.Core/About/about_Comparison_Operators.md

```diff
@@ -377,12 +377,30 @@ Collection examples:
 #Output: Bag, Beg
 ```
 
-`-match` and `-notmatch` support regex capture groups. Each time they run, they
-overwrite the `$Matches` automatic variable. When `<input>` is a collection the
-`$Matches` variable is `$null`. `$Matches` is a **Hashtable** that always has a
-key named '0', which stores the entire match. If the regular expression
-contains capture groups, the `$Matches` contains additional keys for each
-group.
+`-match` and `-notmatch` support regex capture groups. Each time they run on
+scalar input, and the `-match` result is True, or the `-notmatch` result is False,
```

Comment:
In other areas of this document, `True` and `False` are formatted with double asterisks, like `**True**` and `**False**`. Is that something that should be followed in this part of the document as well?

Reply:
This document isn't completely consistent in how it represents True/False (output representation aside).  But it does appear the majority of usage bolds the text.  I'll update this PR to do that.